### ISSUE TITLE
Target profile page rewrite: Update the favicon

### ISSUE THREAD

**Issue by peatroot** [opentargets/platform#337]
We currently use the icon below as the favicon (which appears beside the page title in your browser tab).

![image](https://user-images.githubusercontent.com/7376272/49072910-374a5780-f229-11e8-8fab-8b538984f15a.png)

@andrewhercules, do you want us to use the same one? Currently, the pale blue parts don't show up very well against the grey tabs in chrome. Perhaps it could all be in the darker royal blue that is used? Or white inside a royal blue circle? By royal blue, I mean `#3A8CCB`.

**Comment by andrewhercules** [comment]
Good spot @peatroot! It's really not that visible on the grey tabs in Chrome.

In keeping with an expanded brand palette, here are the favicons I'm proposing for our three main services:

**Open Targets Platform**
![favicon-96x96](https://user-images.githubusercontent.com/7490258/49076616-58af4180-f231-11e8-80d2-531030cdce56.png)

**Open Targets Genetics Portal**
![favicon-96x96](https://user-images.githubusercontent.com/7490258/49076661-6d8bd500-f231-11e8-91cc-42b4ee6457fd.png)

**Open Targets Corporate**
![favicon-96x96](https://user-images.githubusercontent.com/7490258/49076759-a75cdb80-f231-11e8-86ae-89f8ef2c7e37.png)

My approach is based on wanting all three to incorporate the new OT blue `#3A8CCB`, but to also have distinct colours for their respective entities. @LucaFumis, from a design perspective, what do you think?

In [the proof-of-concept folder on the OT Core Platform Team Google Drive](https://drive.google.com/drive/folders/1A7-gVWAIfCs0aiuDQNVL1QGk7YawhzMS), I have uploaded the new Platform favicons in various formats. Can you please try these and let me know if they work and if you need any other formats?

**Comment by deniseOme** [comment]
Cool! Shall we extend this to the docs as well.

The genetics docs seems to have a different favicon when compared to the portal itself.

![screen shot 2018-11-27 at 10 52 41](https://user-images.githubusercontent.com/6472381/49077212-a5dfe300-f232-11e8-9a5a-f2cbfd6fa5a9.png)

**Comment by peatroot** [comment]
Ok, sounds good to me.

**Comment by andrewhercules** [comment]
Yes @deniseOme, great idea! @peatroot, can we use the same favicon for the Platform Documentation site as well?

**Comment by peatroot** [comment]
Yes, I think so.
